src/library_pthread.js

```diff
@@ -311,7 +304,7 @@ var LibraryPThread = {
         worker.on('error', function(e) {
           worker.onerror(e);
         });
-        worker.on('detachedExit', function() {
+        worker.on('threadDone', function() {
```

Comment:
```suggestion
        worker.on('exit', function() {
```
See: https://github.com/emscripten-core/emscripten/pull/12861#discussion_r700102217.